samcli/commands/build/command.py

```diff
@@ -6,6 +6,7 @@
 import logging
 from typing import List, Optional, Dict, Tuple
 import click
+from pathvalidate import ValidationError, validate_filepath
```

Comment:
This seems like a new library licensed from MIT, I would recommend to have our own logic of path verification. To use this library would require some security discussions and at least some extra actions like pinning down a dependency version in the requirements file. If possible let's have a simple method for path validation and catch logic!